CONTRIBUTING.md

```diff
@@ -229,6 +229,16 @@ if $PKG_CONFIG libmongoc-1.0 --atleast-version 1.15.0; then
 AC_MSG_ERROR(system libmongoc must be upgraded to version >= 1.15.0)
 ```
 
+### Update tested versions in evergreen configuration
+
+Evergreen tests against multiple versions of libmongoc. When updating to a newer
+libmongoc version, make sure to update the `libmongoc-version` build axis in
+`.evergreen/config.yml`. In general, we test against two additional versions of
+libmongoc:
+- The upcoming patch release of the current libmongoc minor version (e.g. the
+  `r1.x` branch)
```

Comment:
I suppose this is an argument for not simply _adding_ r1.16 to the axis, but rather replacing the previous entry for r1.15.

Reply:
Yes and no. Let's say 1.16 is released but we don't require it yet (for whatever reason). In this case, it would make sense adding r1.16 to the axis instead of replacing r1.15. Once we require 1.16 or newer, we can drop r1.15 from the axis.